src/bo4e/version.py

```diff
@@ -2,9 +2,12 @@
 Version information for the bo4e package.
 """
 import re
-from importlib.metadata import version
+from importlib.metadata import PackageNotFoundError, version
 
-__version__ = version("bo4e")
+try:
+    __version__ = version("bo4e")
+except PackageNotFoundError:
+    __version__ = "0.0.0"
```

Comment:
This is necessary because we have no version given if the commit has no semantic version in the git tag.

We got the following error message on readthedocs
```
  File "/home/docs/.asdf/installs/python/3.11.6/lib/python3.11/importlib/metadata/__init__.py", line 565, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for bo4e
```
https://readthedocs.org/projects/bo4e-python/builds/22831284/ (unsure how long this link will live ...)